src/main/java/ui/IdGenerator.java

```diff
@@ -19,6 +19,14 @@ public static String getRepositorySelectorId() {
         return "repositorySelector";
     }
 
+    public static String getRepositoryPickerTextFieldId() {
+        return "repositoryPickerUserInputField";
+    }
+
+    public static String getRepositoryPickerSuggestedRepoListId() {
+        return "suggestedRepositoryList";
+    }
+
```

Comment:
Better to place these `RepositoryPicker` related method close to each other? i.e. These two can be brought down to the other ones below.